.github/workflows/publish.yml

```diff
@@ -24,3 +24,16 @@ jobs:
         with:
           token: ${{ secrets.NPM_TOKEN }}
       - run: git push --follow-tags
+
+  Release:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 100
+    - uses: notlmn/release-with-changelog@v1
```

Comment:
Doesn't `notlmn/release-with-changelog` currently depend on the "tag push" event?

https://github.com/notlmn/release-with-changelog/blob/a3b2ce4e86a700ecfd8f13ff9a928afc1fd4dd11/index.js#L13

The tag here is created on line 26, so this has to follow it:

```yml
       - run: git push --follow-tags
       - uses: notlmn/release-with-changelog@v1
         env:
           RELEASE_TOKEN: ${{ secrets.GITHUB_TOKEN }}